pkg/tests/handlers/accounts.go

```diff
@@ -72,6 +101,81 @@ func getAuthorizationToken(connection *sdk.Connection) string {
 	return token
 }
 
+// generateClusterReRegistrationTargeter registers fake clusters and then
+// returns a targeter which uses those fake clusters.
+func generateClusterReRegistrationTargeter(qty int, url string, connection *sdk.Connection) vegeta.Targeter {
+
+	clusterIds := make([]string, qty)
```

Comment:
Why do we want to limit by Quantity? I think this will solve most of my doubts.

Reply:
The purpose of this test is to emulate the following situation:

There are N clusters out in the wild. Roughly every hour, they submit data to telemeter. Telemeter is going to make the request that we're performing in this test to OCM to validate that it should store the data submitted by the Cluster.

I don't really want to just create a bunch of a fake cluster registrations here, since we're already doing that in another performance test. Also, this more closely emulates the test case above as we will be re-using the same set of cluster IDs over and over throughout the duration of this test.